worlds/kh2/Client.py

```diff
@@ -0,0 +1,972 @@
+import os
+import asyncio
+import ModuleUpdate
+import json
+import Utils
+from pymem import pymem
+from worlds.kh2.Items import exclusion_item_table, CheckDupingItems
+from worlds.kh2 import all_locations, item_dictionary_table, exclusion_table
+
+from worlds.kh2.WorldLocations import *
+
+from worlds import network_data_package
+
+if __name__ == "__main__":
```

Comment:
Was this actually tested from the launcher? and by running individually? This line doesn't seem to actually do anything.

Reply:
this line does in fact do nothing but it has been tested by running the .exe and through the launcher.

Reply:
I suspect this is because the launcher is just calling `KH2Client.exe`/`KH2Client.py` which is calling into this file. To fully function as an apworld it should work without that file imo. I don't think that alone is really a blocker atm though.

Reply:
This is using berserker's example with factorio. I dont really know a better solution to this since the client still requires commonclient and that isnt baked into the apworld.

Reply:
You can import CommonClient from apworld without issue, as Factorio does. This is indeed just calling into the KH2Client.py though, might want to follow Factorio closer here with the LauncherComponent additions in __init__.